packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -174,6 +175,9 @@ export default class QuanticSearchInterface extends LightningElement {
     this.urlManager = CoveoHeadless.buildUrlManager(engine, {
       initialState: {fragment: this.fragment},
     });
+    // Make sure to re-synchronize the search interface when the component gets disconnected and reconnected again.
+    this.urlManager.synchronize(this.fragment);
```

Comment:
yeah I tried adding the synchronize here, but I was missing the `this.initialized = false` in disconnectedCallback for this to work properly